test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -799,8 +683,10 @@ func startRealMasterOrDie(t *testing.T, certDir string) (*allClient, clientv3.KV
 		t.Fatal(err)
 	}
 
-	kubeClientConfig := kubeClientConfigValue.Load().(*restclient.Config)
 	storageConfig := storageConfigValue.Load().(storagebackend.Config)
+	kubeClientConfig := restclient.CopyConfig(kubeClientConfigValue.Load().(*restclient.Config))
+	kubeClientConfig.QPS = 99999
```

Comment:
![image](https://media.giphy.com/media/MvedbKot538WY/giphy.gif)

Reply:
five 9's!  It took the test from multi-minute to less than a minute